valgrind_output_compactor.awk

```diff
@@ -0,0 +1,59 @@
+# Downloaded from https://wiki.wxwidgets.org/Valgrind_Suppression_File_Howto on 2019/10/24
+
+#! /usr/bin/awk -f
+# A script to extract the actual suppression info from the output of (for example) valgrind --leak-check=full --show-reachable=yes --error-limit=no --gen-suppressions=all ./minimal
```

Comment:
Line-break after "for example" would help readability on narrow terminals... and github comparison panes :)